__tests__/test.sh

```diff
@@ -0,0 +1,13 @@
+PROJECT_ROOT=$(dirname $0)/../
+cd ${PROJECT_ROOT}
+
+tempdir=test-fixtures/temp
+rm -Rf ${tempdir}
+mkdir ${tempdir}
+
+for i18n in test-fixtures/i18n-*; do
+    out=${tempdir}/$(basename ${i18n}).ts
+    yarn start --i ${i18n} --o ${out} --default en
```

Comment:
`-i` and `-o` should be enough instead of `--i` and `--o`.

Reply:
babel-node has `-i` and `-o` option, so we cannot use them
https://babeljs.io/docs/en/next/babel-node.html#options
Any ideas of other option names?